res/database/Default/database_people.xml

```diff
@@ -257,1017 +276,1295 @@
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<details job="2" gender="0" birthday="" deathday="" country="" fictional="1" />
 			<data popularity="0" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="cd3a2e20-efe1-4a4c-9249-164b67d185f7" tmdb_id="0" imdb_id="nm0000216" creator="">
 			<first_name>Arnold</first_name>
 			<last_name>Weissenacker</last_name>
 			<nick_name>Arni</nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Arnold</first_name_original>
+			<last_name_original>Schwarzenegger</last_name_original>
+			<details job="2" gender="0" birthday="1947-07-30" deathday="" country="" />
 		</person>
 		<person id="ec20fc5a-a552-4119-8d1e-d310650c2830" tmdb_id="0" imdb_id="nm0000142" creator="">
 			<first_name>Klint</first_name>
 			<last_name>Westforest</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Clint</first_name_original>
+			<last_name_original>Eastwood</last_name_original>
+			<details job="2" gender="0" birthday="1930-05-31" deathday="" country="" />
 		</person>
 		<person id="8fd0eba6-3ac0-41eb-8a86-514fa77dc9a2" tmdb_id="0" imdb_id="nm0000080" creator="">
 			<first_name>Orson</first_name>
 			<last_name>Wieles</last_name>
 			<nick_name></nick_name>
-			<details job="1" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Orson</first_name_original>
+			<last_name_original>Welles</last_name_original>
+			<details job="1" gender="0" birthday="1915-05-06" deathday="1985-10-10" country="" />
 		</person>
 		<person id="2b6af8c4-7775-44da-820e-1ce9c55d8cd9" tmdb_id="0" imdb_id="nm0000246" creator="">
 			<first_name>Brus</first_name>
 			<last_name>Will-Es</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Bruce</first_name_original>
+			<last_name_original>Willis</last_name_original>
+			<details job="2" gender="0" birthday="1955-03-19" deathday="" country="" />
 		</person>
 		<person id="f7914690-3554-4d88-bc0d-22be4b48c12f" tmdb_id="0" imdb_id="nm0935498" creator="">
 			<first_name>Michael</first_name>
 			<last_name>Wimslow</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Michael</first_name_original>
+			<last_name_original>Winslow</last_name_original>
+			<details job="2" gender="0" birthday="1958-11-06" deathday="" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="55c09b06-cbe0-4fd9-bfb8-c8ebfb5e1b4e" tmdb_id="0" imdb_id="nm0001845" creator="">
 			<first_name>Forest</first_name>
```

Comment:
Original Name not changed at all

Reply:
Wimslow vs Winslow ... not much Levensthein distance, but still `;)`